repos/fdbt-site/src/pages/selectCaps.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { ReactElement } from 'react';
+import { FullColumnLayout } from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import { ErrorInfo, NextPageContextWithSession, CapInfo } from '../interfaces';
+import CsrfForm from '../components/CsrfForm';
+import { getAndValidateNoc, getCsrfToken } from '../utils';
+import { CapCardBody } from './viewCaps';
+import BackButton from '../components/BackButton';
+import { getCaps } from 'src/data/auroradb';
+import { getSessionAttribute } from 'src/utils/sessions';
+import { CAPS_DEFINITION_ATTRIBUTE } from 'src/constants/attributes';
+
+const title = 'Select Caps - Create Fares Data Service';
+const description = 'Select Caps page of the Create Fares Data Service';
+
+interface SelectCapsProps {
+    csrfToken: string;
+    errors: ErrorInfo[];
+    caps: CapInfo[];
+    backHref: string;
+    selectedId: number | null;
+}
+
+const SelectCaps = ({ csrfToken, errors, caps, backHref, selectedId }: SelectCapsProps): ReactElement => {
+    return (
+        <FullColumnLayout title={title} description={description} errors={errors}>
+            {!!backHref && errors.length === 0 ? <BackButton href={backHref} /> : null}
+            <ErrorSummary errors={errors} />
+            <CsrfForm action="/api/defineCaps" method="post" csrfToken={csrfToken}>
+                <>
+                    <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                        <fieldset className="govuk-fieldset" aria-describedby="contact-hint">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                                <h1 className="govuk-fieldset__heading">Does a cap apply to these product(s)?</h1>
```

Comment:
Ive just realised this only happens for singles, returns and flat fares, all of which are single product journeys. So we can change this text to be "Does a cap apply to this product?"